tracking/src/kld_adaptive_particle_filter.cpp

```diff
@@ -41,10 +41,21 @@
 #include <pcl/impl/instantiate.hpp>
 #include <pcl/point_types.h>
 #define PCL_TRACKING_NORMAL_SUPPORTED
-PCL_INSTANTIATE_PRODUCT(KLDAdaptiveParticleFilterTracker, ((pcl::PointNormal) (pcl::PointXYZINormal) (pcl::PointXYZRGBNormal))(PCL_STATE_POINT_TYPES))
-PCL_INSTANTIATE_PRODUCT(KLDAdaptiveParticleFilterOMPTracker, ((pcl::PointNormal) (pcl::PointXYZINormal) (pcl::PointXYZRGBNormal))(PCL_STATE_POINT_TYPES))
+PCL_INSTANTIATE_PRODUCT(KLDAdaptiveParticleFilterTracker,
```

Comment:
We need to do something about these template instantiations. I don't mean to say that the old formatting was beautiful, but the new one is certainly terrible.

Reply:
```
PCL_INSTANTIATE_PRODUCT(KLDAdaptiveParticleFilterTracker,
                        ((pcl::PointNormal)(pcl::PointXYZINormal)
                         (pcl::PointXYZRGBNormal))(PCL_STATE_POINT_TYPES))
```
Something like this?
If not, please suggest something.

Reply:
```
PCL_INSTANTIATE_PRODUCT(KLDAdaptiveParticleFilterTracker,
                        ((pcl::PointNormal)(pcl::PointXYZINormal)
                         (pcl::PointXYZRGBNormal))
                        (PCL_STATE_POINT_TYPES))
```
Or, this?

Reply:
Addition of `AlignAfterOpenBracket: AlwaysBreak` to the config does this:
```
PCL_INSTANTIATE_PRODUCT(
    KLDAdaptiveParticleFilterTracker,
    ((pcl::PointNormal)(pcl::PointXYZINormal)(pcl::PointXYZRGBNormal))(
        PCL_STATE_POINT_TYPES))
```

Reply:
With `AlignAfterOpenBracket: Align`,
```
PCL_INSTANTIATE_PRODUCT(KLDAdaptiveParticleFilterTracker,
                        ((pcl::PointNormal)(pcl::PointXYZINormal)(
                            pcl::PointXYZRGBNormal))(PCL_STATE_POINT_TYPES))
```